### PR TITLE
Add Litentry Parachain info to Rococo

### PR DESCRIPTION
**Pull request by chenzongxiong**

### CODE REVIEW

File: packages/apps-config/src/endpoints/testingRococo.ts [hunk 1]
```diff
@@ -211,6 +211,14 @@ export function createRococo (t: TFunction): EndpointOption {
         providers: {
           TrustBase: 'wss://rococo.trustednodes.net'
         }
+      },
+      {
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testingRococo.ts:215]: Can you just move this alphabetical? (I know, I let a couple sneak in...)

**Review by jacogr — COMMENTED** [review]
One small comment, looks great otherwise. Thank you.

File: packages/apps-config/src/endpoints/testingRococo.ts (hunk 1 above)
Comment (jacogr) [reply packages/apps-config/src/endpoints/testingRococo.ts:215]: I need to move the rest anyway, will include it there.

**Review by jacogr — APPROVED** [review]

File: packages/apps-config/src/endpoints/testingRococo.ts (hunk 1 above)
Comment (chenzongxiong) [reply packages/apps-config/src/endpoints/testingRococo.ts:215]: I re-ordered all info correctly

**Pull request merged**